### ISSUE TITLE
Question about the training loss

### ISSUE THREAD

**Issue by daiyixiang666** [MAGVIT/magvit#21]
`   vq_codebook_size = get_vq_codebook_size(config)
  num_classes = get_num_classes(config)
  num_special_tokens = get_num_special_tokens(config)
  vocab_size = vq_codebook_size + num_classes + num_special_tokens`
  I found that we not only have the vq_codebook_size and the num_clases and the special_tokens, I 
wonder what are they doing since I do not see they are used in calcualting the loss logits = logits[:, :, :vq_codebook_size]
